<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgSFDCConf</class>
 <widget class="QDialog" name="dlgSFDCConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbAccount">
       <property name="text">
        <string>Your Salesforce.com User ID</string>
       </property>
       <property name="buddy">
        <cstring>eAccount</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="eAccount"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lbPass">
       <property name="text">
        <string>Your password</string>
       </property>
       <property name="buddy">
        <cstring>ePass</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ePass">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbToken">
     <property name="text">
      <string>Security Token (if needed)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="eToken"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>eAccount</tabstop>
  <tabstop>ePass</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
